### DISCUSSION TITLE
How do I set the dollar bill on fire in the Circuit Construction Kit: DC?

### DISCUSSION THREAD

**Discussion by sherbert069** [phetsims/community#29] [Q&A] [answered]
I have been trying for multiple days to figure out how to set it on fire. I've used the most complex systems and the least complex systems to overload the dollar bill. Nothing works, can someone help me out here?

**Comment by veillette** [comment, marked as answer]
According to model.md in circuit-construction-kit-common, 

```
## Household Items

For the purposes of the simulation, we estimate the household item resistances as:

* Coin - 0 Ohms
* Dollar bill - 1,000,000,000 Ohms
* Eraser - 1,000,000,000 Ohms
* Paper clip - 0 Ohms
* Pencil - 25 Ohms
```
A resistance of 1 billion Ohms is extremely high, making it challenging to pass a significant amount of current through it. Even if you employ the high-voltage battery operating at its maximum capacity of 100,000 Volts, the resulting current would be less than one ten-thousandth of an Ampere. While I'm uncertain of the precise current threshold required to ignite objects, the current in this scenario is evidently too minuscule to pass this threshold.
